src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.jsx

```diff
@@ -1,29 +1,20 @@
+import React, { useEffect, useState } from 'react';
 import GordonLoader from 'components/Loader';
 import Moment from 'moment';
-import { Component, Fragment } from 'react';
+import { Fragment } from 'react';
 import { Calendar, momentLocalizer } from 'react-big-calendar';
-import myscheduleService from 'services/myschedule';
 import scheduleService from 'services/schedule';
 import session from 'services/session';
-// @TODO CSSMODULES - Schedule Calendar needs work but left as normal for now
 import './ScheduleCalendar.css';
 
-export default class GordonScheduleCalendar extends Component {
-  constructor(props) {
-    super(props);
+const GordonScheduleCalendar = (props) => {
+  const [loading, setLoading] = useState(true);
+  const [selectedEvent, setSelectedEvent] = useState(null);
```

Comment:
```suggestion
  const [selectedEvent, setSelectedEvent] = useState();
```
useState defaults null

Reply:
It looks like this was resolved by not accepted.  Please either accept the suggestion or explain why not (or point out where I'm missing the change.)

Reply:
I think we might have confused another error for this and marked it as resolved, we have correctly changed it to the suggestion.

Reply:
We got confused and changed the schedule panel index instead of the schedule calendar, both files have been changed.